_support/atix.md

```diff
@@ -0,0 +1,7 @@
+---
+logo: sponsor-atix.png
```

Comment:
Is ATIX sponsoring? I though we only wanted to list them as a party that can provide support. IMHO we should provide a clear separation between sponsors and non-sponsors.

Reply:
Yes. They will be listed at the support page, not the sponsoring page (this is at /_support/, not _sponsors). But the Makefile that generates the png files currently requires all logos to be prefixed with `sponsor-`.

Reply:
Perhaps introduce a new `logo-%.png` rule?

Reply:
updated it